pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -397,7 +404,9 @@ func (r *ReconcileOperConfig) Reconcile(ctx context.Context, request reconcile.R
 		}
 	}
 
-	r.status.SetNotDegraded(statusmanager.OperatorConfig)
+	if !bootstrapResult.OVN.OVNKubernetesConfig.HypershiftConfig.Enabled || bootstrapResult.OVN.OVNKubernetesConfig.HypershiftConfig.Route != "" {
+		r.status.SetNotDegraded(statusmanager.OperatorConfig)
```

Comment:
Presumably this should be SetNotProgressing...

Reply:
I was thinking `SetNotProgressing` might sound that the operator config is not progressing.
By SetNotDegraded, it will override the progressing operator config, right?

Reply:
Is `SetProgressing/UnsetProgressing` more clear in that context?

Reply:
changed to `SetProgressing/UnsetProgressing`.